jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -175,7 +181,7 @@ def step020_install_server(SKIP_SERVER_INSTALL, script_dir, server_era) {
             withEnv(["PUPPET_SERVER_SERVICE_NAME=${server_era["service_name"]}",
                      "PUPPET_SERVER_TK_AUTH=${server_era["tk_auth"]}",
                      "PACKAGE_BUILD_VERSION=${server_era["version"]}",
-                     "PUPPET_AGENT_VERSION=${server_era["agent_version"]}"]) {
+                     "PUPPET_AGENT_VERSION=${agent_version}"]) {
```

Comment:
Might just be my absent knowledge of Groovy, not sure.  I was expecting that for the value received for `agent_version` on line 386 that you would need to pass it through as a parameter on a call to `step020_install_server` on line 389 so that the value of `agent_version` on this line would be resolvable.  Maybe it works without doing that but it would seem more obvious if it were passed through as a parameter to `step020_install_server`, like `server_era` is.  Does that make sense?